sessionctx/variable/sysvar.go

```diff
@@ -456,6 +456,12 @@ var defaultSysVars = []*SysVar{
 			return BoolToOnOff(config.GetGlobalConfig().Instance.TiDBEnableDDL.Load()), nil
 		},
 	},
+	{Scope: ScopeInstance, Name: TiDBRCReadCheckTS, Value: BoolToOnOff(DefRCReadCheckTS), Type: TypeBool, SetGlobal: func(s *SessionVars, val string) error {
```

Comment:
Should the `Value` be set to the instance config in global config?

Reply:
In fact, I added this instance variable referring to tidb_general_log. It looks simple.
um, what's the difference of this way and global config?

Reply:
I see, it's actually set in `setGlobalVars`. The whole mechanism looks not so consistent, though...